test/metabase/api/persist_test.clj

```diff
@@ -43,3 +45,39 @@
         (is (= default-cron
                (get-in (task.persist-refresh/job-info-by-db-id)
                        [(:id db) :schedule])))))))
+
+(deftest persisted-info-by-id-test
+  (with-setup db
+    (t2.with-temp/with-temp
+      [:model/Card          model {:database_id (u/the-id db), :dataset true}
+       :model/PersistedInfo pinfo {:database_id (u/the-id db), :card_id (u/the-id model)}]
+      (testing "Should require a non-negative persisted-info-id"
+        (is (= "API endpoint does not exist."
+               (mt/user-http-request :crowberto :get 404 (format "persist/%d" -1)))))
+      (testing "Should not get info when the persisted-info-id doesn't exist"
+        (is (= "Not found."
+               (mt/user-http-request :crowberto :get 404 (format "persist/%d" Integer/MAX_VALUE)))))
+      (testing "Should get info when the ID exists"
+        (is (=? {:active true
+                  :card_id (u/the-id model)
+                  :id (u/the-id pinfo)
+                  :state "persisted"}
```

Comment:
```suggestion
        (is (=? {:active  true
                 :card_id (u/the-id model)
                 :id      (u/the-id pinfo)
                 :state   "persisted"}
```

nit but we tried to align maps in this way too

Reply:
Thanks, committed in https://github.com/metabase/metabase/pull/33776/commits/653af6c6fb3d7533d9b64e1bbaca0beee55536d9